README.md

```diff
@@ -214,6 +214,16 @@ These extensions are good complements to PaperWM:
 - [Switcher](https://github.com/daniellandau/switcher) - combined window switcher and launcher
 - [Dash to Dock](https://micheleg.github.io/dash-to-dock/) - a great dock
 
+## Note for gnome users ##
+
+If desktop-icon is enabled (which it is by default), clicking anywhere while on the stack view will cause windows to not take any user inputs. If this happens log out the user session to fix it.
```

Comment:
Since the exact issues might change over time (and is not limited to #145) something like: "The desktop-icon extension (on by default) and PaperWM are not compatible. Eg.: issue #145." might be better?

Reply:
That makes sense